ios-base/DroidKaigi 2020/About/Views/AboutViewController.swift

```diff
@@ -0,0 +1,165 @@
+import ios_combined
+import SafariServices
+import UIKit
+
+class AboutViewController: UITableViewController {
+    private enum CellType {
+        case description, access, staff, policy, license, version
+    }
+
+    private let cells: [CellType] = [.description, .access, .staff, .policy, .license, .version]
+
+    static func instantiate() -> AboutViewController {
+        guard let viewController = UIStoryboard(name: "AboutViewController", bundle: .main).instantiateInitialViewController() as? AboutViewController else { fatalError() }
+        return viewController
+    }
+
+    override func viewDidLoad() {
+        title = L10n.aboutDroidKaigi
+    }
+}
+
+extension AboutViewController {
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        cells.count
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        switch cells[indexPath.row] {
+        case .description:
+            return dequeueReusableCell(withIdentifier: AboutCell.Identifier.description, for: indexPath) { cell in
+                cell.titleLabel.text = L10n.whatIsDroidKaigi
+                let detail = L10n.droidKaigiDescription
+                let paragraphStyle = NSMutableParagraphStyle()
+                paragraphStyle.lineHeightMultiple = 1.1
+                let attributes: [NSAttributedString.Key: Any] = [.kern: 0.5, .paragraphStyle: paragraphStyle]
+                cell.detailLabel?.attributedText = NSMutableAttributedString(string: detail,
+                                                                             attributes: attributes)
+                cell.onButtonTapped = { [weak self] button in
+                    guard let self = self else { return }
+                    let urlString: String
+                    switch button {
+                    case .twitter:
+                        urlString = "https://twitter.com/DroidKaigi"
+                    case .youtube:
+                        urlString = "https://www.youtube.com/channel/UCgK6L-PKx2OZBuhrQ6mmQZw"
+                    case .medium:
+                        urlString = "https://medium.com/droidkaigi"
+                    }
+                    guard let url = URL(string: urlString) else { return }
+                    self.presentSafariViewController(with: url)
```

Comment:
How about open URL with `UIApplication.shared.open(url)` ? 🤔  
It can open with Twitter App, YouTube App, and Medium App.

Reply:
You are right.
I have changed to `UIApplication.shared.open(url)`.